### PR TITLE
Setup and  Mobile finish

### PR DESCRIPTION
**Pull request by pmaxy**
Things done

- Add work and about section
- error checked and commit work

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,13 +26,153 @@ <h1>I’m Patrick
...
+			<h2>About Myself</h2>
+			<p>
+				Hello I’m a software developer! 
+				I can help you build a product ,
+				feature or website Look through
+				some of my work and experience!
+				If you like what you see and have a project you need coded, 
+				don’t hestiate to contact me.
+			</p>
+			<a href="">LET’S CONNECT</a>
```
Comment (didierganthier) [inline index.html]: Well done on the design, it looks fire 🔥 but talking about semantics, we can improve this a bit more. The title LET'S CONNECT is very important in your portfolio and it needs to be better referenced by browsers 🌐 so in order to achieve that consider using a header tag `hx` so you could have something like below:
 ```
 <h3>LET’S CONNECT</h3>
 ```
 I suppose that also means editing your CSS 🤔  so make sure to edit it or you can also do the following:
 ```
  <h3><a href="">LET’S CONNECT</a></h3>
  ```
This implies of course that you have no styles predefined for `h3` tags, I hope this helps 😉

File: index.html [hunk 2]
```diff
@@ -25,13 +26,153 @@ <h1>I’m Patrick
...
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
```
Comment (didierganthier) [inline index.html:122]: This is a nice button that you created here 💯 but in order to match the Figma a little bit more and to make your project cooler 😎 The button should follow the button styling: Active, Hover and Pressed [see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=WRpq1TpJZoIQMeK1-0). Take a look below:

Screenshot 2023-01-11 at 20 26 03

File: index.html [hunk 3]
```diff
@@ -25,13 +26,153 @@ <h1>I’m Patrick
...
+			</p>
+			<a href="">LET’S CONNECT</a>
+			<ul class="tags">
+				<li><a href=""><img src="/img/github.svg" alt=""></a></li>
+				<li><a href=""><img src="/img/Linkedin.svg" alt=""></a></li>
+				<li><a href=""><img src="/img/medium.svg" alt=""></a></li>
+				<li><a href=""><img src="/img/twiter.svg" alt=""></a></li>
+				<li><a href=""><img src="img/chill.svg" alt=""></a></li>
+			</ul>
+			<button class="btn">Get my resume</button>
```
Comment (didierganthier) [inline index.html:145]: Same thing I mentioned earlier for this button 👍🏽  the button should follow the button styling: Active, Hover and Pressed [see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=WRpq1TpJZoIQMeK1-0). Take a look below:

Screenshot 2023-01-11 at 20 26 03

Your CSS would have to include something like this:
```
.btn:hover {
   // your cool style here
}
```
Best of luck with the rest 🥇

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @pmaxy , 

Good job so far! 💯 
There are some issues that you still need to work on to go to the next project but you are almost there! You did a great job on the design and I believe you will do wonder at Microverse, keep going. 🔥 



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct Github flow ✔️ 
- Nice UI design ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @pmaxy    :wave: 
### I'm Levy, your PR reviewer

 Excellent, you nailed it :sunglasses: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 



## Highlights
- Your README file is professional :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**